oapi-config.yaml

```diff
@@ -0,0 +1,5 @@
+generate:
+  models: true
+  gin-server: true
+  strict-server: true
+package: foo
```

Comment:
Does this package value do anything?

Reply:
No, and in fact that file isn't even used now. Removing it.